frameworks/Perl/web-simple/app.pl

```diff
@@ -3,7 +3,7 @@
 use JSON::XS;
 use DBI;
 
-my $dsn = "dbi:mysql:database=hello_world;host=localhost";
+my $dsn = "dbi:mysql:database=hello_world;host=127.0.0.1";
```

Comment:
This change seems unnecessary, but if it **is** necessary, then [this line](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Perl/web-simple/setup.sh#L3) will have to be updated accordingly.